src/cli.rs

```diff
@@ -13,4 +20,10 @@ pub struct Cli {
     /// increase verbosity.
     #[arg(short, long, action = clap::ArgAction::Count)]
     pub verbose: u8,
+
+    /// The path where to create a listening Unix domain socket.
+    ///
+    /// This is the path where Cloud Hypervisor will connect to usbvfiod.
+    #[arg(short, long)]
+    pub socket_path: PathBuf,
```

Comment:
nit: we are leaving out the `fd` option on purpose currently?

I prepared the following recently:

```suggestion
    /// Provide the vfio-user socket as file-descriptor.
    #[arg(long, conflicts_with = "socket_path")]
    pub fd: Option<RawFd>,
    /// Provide the vfio-user socket as path.
    #[arg(long, required_unless_present = "fd")]
    pub socket_path: Option<PathBuf>,
```

Reply:
We agreed to implement the command line option and trigger a `todo!` when someone uses it.